MapboxNavigation/UIView.swift

```diff
@@ -70,8 +70,8 @@ extension UIView {
     }
     
 
-    class func forAutoLayout<ViewType: UIView>() -> ViewType {
-        let view = ViewType.init(frame: .zero)
+    class func forAutoLayout<ViewType: UIView>(frame: CGRect = .zero) -> ViewType {
```

Comment:
Suggestion: Add a hidden argument as well, e.g: `forAutoLayout<ViewType: UIView>(frame: CGRect = .zero, hidden: Bool = false)`. That way, you can avoid having to break out `lanesView` and `nextBannerView` into closures.